.css

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+body {
+  background-color: pink;
+}
+
+h1 {
+  color: purple;
+  text-align: center;
+}
+
+</style>
+</head>
+<body>
+
+<h1>Hello Microverse!</h1>
+
+
+</body>
+</html>
+
```

Comment:
Great job @TayyabKM 🤝

- [ ] Kindly remove markup from your stylesheet as this file should contain styles being applied to the markup in the HTML file.
- [ ] Also remember to give a descriptive name to your CSS file. ie `styles.css`.